packages/@react-aria/utils/test/number.test.js

```diff
@@ -4,17 +4,17 @@ import {clamp, snapValueToStep} from '../src/number';
 describe('number utils', () => {
   describe('clamp', () => {
     it('should constrain value between min and max', () => {
-      assert.equal(clamp(5, -1, 1), 1);
-      assert.equal(clamp(-5, -1, 1), -1);
+      assert.strictEqual(clamp(5, -1, 1), 1);
```

Comment:
these are jest, they should probably be moved to expects
`expect(clamp(5, -1, 1)).toBe(1)`